src/components/footer/_macro.njk

```diff
@@ -94,6 +94,10 @@
                         </div>
                     {% endif %}
 
+                    {% if params.newTabWarning is defined and params.newTabWarning %}
+                        <p class="u-fs-s footer__new-tab-warning">{{ params.newTabWarning }}</p>
```

Comment:
If you wrap this in `grid__col` and add `u-mb-m` to the `<p>`, you don't need any css.